src/app/classes/category.ts

```diff
@@ -0,0 +1,150 @@
+export class Category {
+
+  private category: string;
+  private subcategory: string;
+  private categoryFE: string;
+
+  public Category() {};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7556402)
```suggestion
  public Category() {}
```